docs/cudf/source/user_guide/data-types.md

```diff
@@ -0,0 +1,156 @@
+# Supported Data Types
+
+cuDF supports many data types supported by NumPy and Pandas, including
+numeric, datetime, timedelta, categorical and string data types. We
+also provide special data types for working with decimals, list-like,
+and dictionary-like data.
+
+All data types in cuDF are [nullable](/user_guide/missing-data).
+
+<div class="special-table">
+
+| Kind of data         | Data type(s)                                                                    |
+|----------------------|---------------------------------------------------------------------------------|
+| Signed integer       | `'int8'`, `'int16'`, `'int32'`, `'int64'`                                       |
+| Unsigned integer     | `'uint32'`, `'uint64'`                                                          |
+| Floating-point       | `'float32'`, `'float64'`                                                        |
+| Datetime             | `'datetime64[s]'`, `'datetime64[ms]'`, `'datetime64['us']`, `'datetime64[ns]'`  |
+| Timedelta (duration) | `'timedelta[s]'`, `'timedelta[ms]'`, `'timedelta['us']`, `'timedelta[ns]'`      |
+| Category             | `cudf.CategoricalDtype`                                                         |
+| String               | `'object'` or `'string'`                                                        |
+| Decimal              | `cudf.Decimal32Dtype`, `cudf.Decimal64Dtype`, `cudf.Decimal64Dtype`             |
+| List                 | `cudf.ListDtype`                                                                |
+| Struct               | `cudf.StructDtype`                                                              |
+
+</div>
+
+## NumPy data types
+
+We use NumPy data types for integer, floating, datetime, timedelta,
+and string data types.  Thus, just like in NumPy,
```

Comment:
This is just Emacs'Emacs' `auto-fill` at work. Personally, I think [Semantic Linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) are the best convention for documentation source, but it's hard to incorporate them in anything but personal projects :(

Reply:
Vim’s “gq” does the same thing with two spaces. I would be open to rewriting this with semantic lines (or the easier to enforce “one sentence per line”). I have had no issues in practice with using that style on other projects. This word wrap is pretty harsh right now - it’s wrapping at less than 80 chars (74?), so I think it is unlikely that this word wrapped formatting will survive more than one or two revisions anyway.

Reply:
Perhaps a discussion to have offline/as part of the dev guide?

Reply:
I would not oppose making that change in this PR, since all the files are being rewritten anyway. If you don't want to make that decision in this PR, this would be a good topic for tomorrow's cuDF Python meeting.

Reply:
+1 for semantic linefeeds